### ISSUE TITLE
Content check looking for OG tags

### ISSUE THREAD

**Issue by marcusosterberg** [Webperf-se/webperf_core#122]
Can we argue the usability of a page when shared on social media that OpenGraph is essential? Just as meta description is mandatory in SEO for all important pages. 

https://neilpatel.com/blog/open-graph-meta-tags/

**Comment by 7h3Rabbit** [comment]
@marcusosterberg Please specify what tags you want to test for and also if you have any ideas on how to score them

**Comment by 7h3Rabbit** [comment]
I think this should be a new test and not added to a existing test.
Is there anything more that might be related?

**Comment by marcusosterberg** [comment]
Yes, in that case I think we should consider another web standards test focused on *structured data*. 

Checking for:
1. DCAT-AP to consume by [data catalogues](https://docs.dataportal.se/dcat/sv/)
2. [Schema.org](https://schema.org/), if possible both microdata and JSON-LD (RDFa might be a bit of a stretch?)
3. [OpenGraph](https://ogp.me/) is commonly used but not that much of a open standard

**Comment by cockroacher** [comment]
@marcusosterberg I am not fluent in DCAT-AP. What do you think should be validated?

**Comment by marcusosterberg** [comment]
@cockroacher, never mind. I thought there were some standardised way to discover DCAT information on a website.

For Schema.org it's worth looking if it is present at all. 
The content Types I think is interesting to look for specifically is:
* https://schema.org/Blog
* https://schema.org/BlogPosting
* https://schema.org/Article
* https://schema.org/NewsArticle
* https://schema.org/MediaObject

Also, who is talking:
* https://schema.org/author
* https://schema.org/Organization - both present as contact info and as a generic author
* https://schema.org/Person

Similar to OpenGraph there are:
* https://schema.org/WebPage that can list its primary image https://schema.org/ImageObject

Structured metadata such as:
* https://schema.org/WebSite
* https://schema.org/BreadcrumbList

I'm not sure we should try to validate the code, but there is a public validation at https://validator.schema.org that is looking at the syntax. For people more interested in how Google interprets the code there is also https://search.google.com/test/rich-results

**Comment by marcusosterberg** [comment]
Perhaps we should look for a meta-description as well?
